source/guide_taskd.rst

```diff
@@ -0,0 +1,370 @@
+.. author:: seeq
+
+.. tag:: activity-tracking
+.. tag:: console
+.. tag:: sync
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/taskwarrior.png
+      :align: center
+
+########
+Taskd
+########
+
+.. tag_list::
+
+Taskd_ is a Free and Open-Source Taskwarrior Server for synchronization between multiple Taskwarrior clients.
+Taskwarrior is Free and Open Source Software that manages your TODO list from the command line. It is flexible, fast, and unobtrusive. It does its job then gets out of your way. 
+
+* Multiple clients
+* TLS secured
+* Andorid App available (also in FDroid)
+* Is MIT-licensed free software.
+
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+  * The taskwarrior application which is available in almost all packet managers. On Android look for the taskwarrior app (also available at FDroid).
+
+Prerequisites
+=============
+
+Before you sync anything, make a backup of your local task data. You never know what might happen.
+::
+
+ [isabell@othermachine ~]$ tar czf task-backup-$(date +'%Y%m%d').tar.gz *
+ [isabell@othermachine ~]$
+
+Move this file somewhere safe.
+
+Your taskd domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Taskd listens on a port (by default ``53589``). We need to open a port in the firewall that our client can connect to the task server.
+
+::
+
+ [isabell@stardust ~]$ uberspace port add
+ Port 40200 will be open for TCP and UDP treffic in a few minutes.
+ [isabell@stardust ~]$
+
+remember this port. To connect later, we'll also need to know the hostname:
+
+::
+
+ [isabell@stardust ~]$ hostname -f
+ stardust.uberspace.de
+ [isabell@stardust ~]$
+
+
+We need a directory where the taskd files are stored.
+::
+
+ [isabell@stardust ~]$ export TASKDDATA=/home/isabell/.var/taskddata
+ [isabell@stardust ~]$ mkdir -p $TASKDDATA
+ [isabell@stardust ~]$
+
+The ``$TASKDDATA`` environment variable is expected by the taskdctl binary we are going to install now. 
+
+
+Installation
+============
+
+
+Download Sources
+----------------
+
+Download the latest version and build it.
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ curl -LO https://taskwarrior.org/download/taskd-1.1.0.tar.gz
+  ...
+  [isabell@stardust ~]$ tar xzf taskd-1.1.0.tar.gz
+  [isabell@stardust ~]$ cd taskd-1.1.0
+  [isabell@stardust ~]$ cmake -DCMAKE_BUILD_TYPE=release .
+  ...
+  [isabell@stardust ~]$ make
+  ...
+  [isabell@stardust ~]$
+
+Install Required taskd Binaries
+-------------------------------
+::
+
+ [isabell@stardust ~]$ cp /home/isabell/taskd-1.1.0/src/taskd /home/isabell/bin
+ [isabell@stardust ~]$ cp /home/isabell/taskd-1.1.0/src/taskdctl /home/isabell/bin
+ [isabell@stardust ~]$
+
+Verify installation:
+::
+
+ [isabell@stardust ~]$ taskd -v
+ [isabell@stardust ~]$ taskdctl status
+ /usr/local/bin/taskdctl status: daemon not running
+ [isabell@stardust ~]$
+
+
+Install certtool
```

Comment:
We'll provide gnutls-utils which provides certtool with Uberspace 7.3.7.

Reply:
gnutls-utils is now available: https://manual.uberspace.de/changelog.html#v7-3-7